package.json

```
@@ -84,8 +84,10 @@
     "simpleserver:stop": "kill-port 3000",
     "fmt": "prettier --write ./src *.ts *.json *.js ./e2e ./packages",
     "fmt-check": "prettier --check ./src *.ts *.json *.js ./e2e ./packages",
-    "fetch:wasm": "./get-latest-wasm-bundle.sh",
-    "fetch:samples": "echo \"Fetching latest KCL samples...\" && curl -o public/kcl-samples-manifest-fallback.json https://raw.githubusercontent.com/KittyCAD/kcl-samples/achalmers/kw-pattern/manifest.json",
+    "install:tools:windows": "winget install Schniz.fnm jq yq",
```

```suggestion
    "install:tools:windows": "winget install Schniz.fnm jqlang.jq MikeFarah.yq",
```

You're right @franknoirot, this should work better.